src/main/java/org/spongepowered/api/event/entity/player/PlayerEvent.java

```diff
@@ -38,18 +39,9 @@
      *
      * @return The {@link Player} involved
      */
-    Player getPlayer();
-
-    @Override
-    Player getHuman();
-
-    @Override
-    Player getLiving();
-
     @Override
     Player getEntity();
 
     @Override
-    Player getUser();
-
+    User getUser();
```

Comment:
Thus there is no reason to return anything but user. It still functions the same way, and keeps in line with super classes.